### DISCUSSION TITLE
about Office URI Schemes

### DISCUSSION THREAD

**Discussion by hoshizorayuuta** [MicrosoftDocs/office-developer-client-docs#660] [Q&A]
When using the OFFICE URI to open the word, powerpoint, excel, use "?" to bring the parameters, powerpoint and excel can normally pass the parameters back to the server, but the word will not bring the parameters back.
Word:ms-word:ofe|u|uri?parameters
Powerpoint:ms-powerpoint:ofe|u|uri?parameters
Excel:ms-excel:ofe|u|uri?parameters

before issues:https://github.com/OfficeDev/office-js/issues/729

**Comment by lindalu-MSFT** [comment]
@hoshizorayuuta This is a known issue and is currently on the backlog.
